src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -643,13 +644,13 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
    * @private
    */
   _getParsedAdaptiveVideoTracks(): Array<Track> {
-    //TODO check adaptation in safari hls
     const videoTracks = this._videoElement.videoTracks;
     const parsedTracks = [];
     if (videoTracks) {
       for (let i = 0; i < videoTracks.length; i++) {
         const settings = {
-          //TODO calculate width/height/bandwidth
+          height: this._videoElement.videoHeight,
+          width: this._videoElement.videoWidth,
```

Comment:
this value will change on change of video track so we can't know it upfront like this AFAIK

Reply:
fixed